### PR TITLE
cli: rook tool object connection information output to environment va…

### PR DESCRIPTION
**Pull request by jbw976**
…riables

### CODE REVIEW

File: scripts/clean-rookd.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 sudo rm -fr /var/lib/rook
+sudo mkdir /var/lib/rook;sudo chown -R $USER:$USER /var/lib/rook
```
Comment (travisn) [inline scripts/clean-rookd.sh]: why create the folder after deleting it? is this so it's simpler to start the cluster on the next run?

File: cmd/rook/object/connection.go [hunk 2]
```diff
@@ -67,17 +87,26 @@ func getConnectionInfo(c client.RookRestClient) (string, error) {
...
-	fmt.Fprintf(w, "%s\t%s\t\n", "S3_SECRET_KEY", connInfo.SecretKey)
+		// write object store connection info
+		fmt.Fprintf(w, "%s\t%s\t\n", AWSHost, connInfo.Host)
+		fmt.Fprintf(w, "%s\t%s\t\n", AWSEndpoint, connInfo.IPEndpoint)
+		fmt.Fprintf(w, "%s\t%s\t\n", AWSAccessKey, connInfo.AccessKey)
+		fmt.Fprintf(w, "%s\t%s\t\n", AWSSecretKey, connInfo.SecretKey)
+
+		w.Flush()
+	} else if format == FormatEnvVar {
+		buffer.WriteString(fmt.Sprintf("export %s=%s\n", AWSHost, connInfo.Host))
```
Comment (travisn) [inline cmd/rook/object/connection.go]: nit: declare a const for the export?

**Comment by jbw976** [comment]
Addresses #262

File: cmd/rook/object/connection.go [hunk 3]
```diff
@@ -67,17 +87,26 @@ func getConnectionInfo(c client.RookRestClient) (string, error) {
...
+		// write header columns
+		fmt.Fprintln(w, "NAME\tVALUE")
 
-	// write object store connection info
-	fmt.Fprintf(w, "%s\t%s\t\n", "S3_HOST", connInfo.Host)
-	fmt.Fprintf(w, "%s\t%s\t\n", "S3_ENDPOINT", connInfo.IPEndpoint)
-	fmt.Fprintf(w, "%s\t%s\t\n", "S3_ACCESS_KEY", connInfo.AccessKey)
-	fmt.Fprintf(w, "%s\t%s\t\n", "S3_SECRET_KEY", connInfo.SecretKey)
+		// write object store connection info
+		fmt.Fprintf(w, "%s\t%s\t\n", AWSHost, connInfo.Host)
```
Comment (travisn) [inline cmd/rook/object/connection.go]: const here as well?

File: scripts/clean-rookd.sh (hunk 1 above)
Comment (jbw976) [inline scripts/clean-rookd.sh]: yes, i was doing this as a manual step before starting rookd the next time.   much nicer to remove that manual step from my flow.

File: cmd/rook/object/connection.go (hunk 3 above)
Comment (jbw976) [inline cmd/rook/object/connection.go]: sure, can do for both the recommended consts

**Pull request merged**